pymtl3/passes/translator/test/RTLIRTranslator_L4_cases_test.py

```diff
@@ -15,7 +15,7 @@ def local_do_test( m ):
   if not m._dsl.constructed:
     m.elaborate()
   tr = TestRTLIRTranslator(m)
-  tr.translate()
+  tr.translate( m )
```

Comment:
why do you need this translate(m)?

Reply:
Because these tests under `translator` are before the support for translating non-top components is added. After I added this support,  `Translator.__init__` now takes the top component as a parameter and `translator.translate` is a called with the actual component you want to translate.